src/routes/popup/agents/liquidops/utils/LiquidOps.ts

```diff
@@ -0,0 +1,51 @@
+import LiquidOps from "liquidops";
+import { ArweaveSigner, createData } from "@dha-team/arbundles";
+import { getActiveKeyfile, type DecryptedWallet } from "~wallets";
+import { isLocalWallet } from "~utils/assertions";
+import { freeDecryptedWallet } from "~wallets/encryption";
+import type { JWKInterface } from "arweave/web/lib/wallet";
+
+export const LiquidOpsClient = async (useWallet = false) => {
+  const decryptedWallet: DecryptedWallet = useWallet
```

Comment:
You might want to wrap this in a `try-catch`. Otherwise, `getActiveKeyfile()` might get called but not `freeDecryptedWallet()`, if there's an error.